api/job-batches/params/job-batches-parameters.ts

```diff
@@ -0,0 +1,34 @@
+import { BaseParameters } from "../../parameters";
+import { SmartlingException } from "../../exception";
+
+export class JobBatchesParameters extends BaseParameters {
+    setDescription(description: string): this {
+        if (description.length >= 2000) {
+            throw new SmartlingException("Job description should be less than 2000 characters.");
+        }
+
+        this.set("description", description);
+
+        return this;
+    }
+    setTargetLocaleIds(targetLocaleIds: string[]): this {
+        this.set("targetLocaleIds", targetLocaleIds);
+
+        return this;
+    }
+    setMode(mode: "CREATE_NEW" | "REUSE_EXISTING"): this {
```

Comment:
I'm curious how developer will use these consts in his code. Should he pass as a string? Then regular Enum looks better to me, when I can simply use the enum in my code.

Reply:
Done